docs/installation-guide.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Installation Guide"
+weight: 10
+---
+
+
+There are various ways you can use the Generator library to generate whaterver you want in your event driven architecture apps. Find your prefered method below:
+- [1. Using the AsyncAPI Generator CLI](#1-using-the-asyncapi-generator-cli)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+  - [Updating the AsyncAPI Generator CLI](#updating-the-asyncapi-generator-cli)
+  - [Uninstall the AsyncAPI Generator CLI](#uninstall-the-asyncapi-generator-cli)
+- [2. Use the Generator Library In Your Node.js Apps](#2-use-the-generator-library-in-your-nodejs-apps)
+  - [Prerequisites](#prerequisites-1)
+ 
```

Comment:
@derberg I wanted to link the 2 ways to install the Generator here but VS code kept auto-generating the table of contents instead. I just wanted it to be:

1.  [Using the AsyncAPI Generator CLI](#1-using-the-asyncapi-generator-cli) and 
2.  [Use the Generator Library In Your Node.js Apps](#2-use-the-generator-library-in-your-nodejs-apps)
Not sure how to achieve this.